.github/actions/publish-nuget/action.yml

```diff
@@ -0,0 +1,126 @@
+name: Publish to NuGet
+description: Builds all .NET solutions in the given directory with static code analysis.
+
+inputs:
+  source:
+    required: false
+    default: https://api.nuget.org/v3/index.json
+    description: The NuGet server URL used by the `dotnet nuget push` command's `--source` argument. 
+  verbosity:
+    required: false
+    default: minimal
+    description: The logging verbosity type used by the `dotnet` command.
+secrets:
```

Comment:
VS says this despite it working:
![image](https://user-images.githubusercontent.com/1976647/176326548-0d3b7588-bca2-4cf7-b459-9690fa21ac45.png)

Reply:
I guess the schema used by VS must be out of date. Clearly we need that property there or it will break.